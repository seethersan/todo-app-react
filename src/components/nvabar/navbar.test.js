import React from 'react';
import { shallow, mount, render } from 'enzyme';
import NavBar from './navbar.component';

it('expect to render NavBar component', () => {
    expect(shallow(<NavBar />)).toMatchSnapshot();
});