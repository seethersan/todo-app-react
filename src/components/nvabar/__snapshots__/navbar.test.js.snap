// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render NavBar component 1`] = `ShallowWrapper {}`;
